src/types/interfaces.ts

```diff
@@ -624,26 +653,56 @@ export function isSecurityCode (data: { securityCode: string } | ICustomerData):
   );
 }
 
-export interface IPendingSession {
+export { ISessionInfo };
+export interface IPendingSession extends ISessionInfo {
   id: string;
-  autoAnswer: boolean;
-  address: string;
-  conversationId: string;
   sessionType: SessionTypes;
```

Comment:
As an anecdote to this, we should simplify the creation of the `IPendingSession` in `session-manager.onPropose` since we should basically just pass the `ISessionInfo` through, minus adding the `id` property. Also, the `handlePendingSession` in the base-session-handler should no longer need to add the `sessionType` since streaming client is doing that now.

Reply:
I can update `onPropose` and `handlePendingSession`. The reason I put `sessionType: SessionTypes` here is because the streaming-client defines `type SessionTypes = 'softphone' | 'acd-screenshare' | ...` where the SDK defines it as an `enum`. I was being a little lazy, tbh. I think streaming-client needs to be updated to an `enum` and then we can use that from the SDK.